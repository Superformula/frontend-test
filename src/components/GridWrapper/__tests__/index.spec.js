import React from 'react';
import { mount } from 'enzyme';
import { GridWrapper } from '../';

describe('GridWrapper', () => {
  it('should render without failing', () => {
    const wrapper = mount(<GridWrapper />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render flex grid wrapper', () => {
    const wrapper = mount(<GridWrapper flex />);
    expect(wrapper).toMatchSnapshot();
  });
});
