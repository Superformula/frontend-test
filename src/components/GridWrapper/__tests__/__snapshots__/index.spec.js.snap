// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridWrapper should render flex grid wrapper 1`] = `
.c0 {
  width: 100%;
  padding-right: 24px;
  padding-left: 24px;
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (min-width:758px) {
  .c0 {
    padding-right: 64px;
    padding-left: 64px;
  }
}

<GridWrapper
  flex={true}
>
  <div
    className="c0"
  />
</GridWrapper>
`;

exports[`GridWrapper should render without failing 1`] = `
.c0 {
  width: 100%;
  padding-right: 24px;
  padding-left: 24px;
  position: relative;
}

@media (min-width:758px) {
  .c0 {
    padding-right: 64px;
    padding-left: 64px;
  }
}

<GridWrapper>
  <div
    className="c0"
  />
</GridWrapper>
`;
